### PR TITLE
ahoy 2.0.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by ocean** [comment]
Hi there @Porkepix - I'm a maintainer of `ahoy` and I have changed and cleaned up brew install command slightly in the recipe, can you please close this PR so my updated one is included instead?

https://github.com/Homebrew/homebrew-core/pull/104850

**Comment by chenrui333** [comment]
let's move the discussions into that PR.

**Comment by ocean** [comment]
Hi @chenrui333 argh, I must have just sent this message after you approved it, oh well

**Pull request closed without merging**
